plugins/mcp/main.go

```
@@ -0,0 +1,928 @@
+	return res, nil, nil
+}
+
+func (p *MCPPlugin) Cleanup() error {
+	p.mu.Lock()
+	defer p.mu.Unlock()
+
+	// Clean up STDIO commands
+	for _, client := range p.clientMap {
+		if client.StdioCommand != nil && client.StdioCommand.Process != nil {
+			fmt.Printf("Terminating STDIO process: %d\n", client.StdioCommand.Process.Pid)
+			client.StdioCommand.Process.Kill()
```

golangci-lint (1.64.8)

Error return value of `client.StdioCommand.Process.Kill` is not checked

(errcheck)